### PR TITLE
Add auraBal -> bpt slippage check

### PR DESCRIPTION
**Pull request by shuklaayush**

### CODE REVIEW

File: contracts/MyStrategy.sol [hunk 1]
```diff
@@ -132,6 +137,11 @@ contract MyStrategy is BaseStrategy, ReentrancyGuardUpgradeable {
         }
     }
 
+    function setAuraBalToBalEthBptMinOutBps(uint256 _minOutBps) external {
+        _onlyGovernanceOrStrategist();
```
Comment (sajanrajdev) [inline contracts/MyStrategy.sol:141]: I think we should add a require to ensure that the mount is never above MAX_BPS, otherwise harvests will fail.

**Comment by sajanrajdev** [comment]
Other than the comments from above, the changes LGTM. Adds a slippage check for the balAURA -> BALETH_BPT swap, since these are paired 1:1, it is safe to check for slippage by simply multiplying the tolerance times the in amount.

The slippage tolerance can be changed by gov and strategist.

All tests are passing:
image

File: tests/upgrade/test_upgrade.py [hunk 2]
```diff
@@ -27,34 +29,39 @@ def proxy_admin_gov(proxy_admin):
...
-   
-  assert old_processLocksOnReinvest == strat_proxy.processLocksOnReinvest()
+    assert old_withdrawalSafetyCheck == strat_proxy.withdrawalSafetyCheck()
+    
+    assert old_processLocksOnReinvest == strat_proxy.processLocksOnReinvest()
 
-  assert old_bribesProcessor == strat_proxy.bribesProcessor()
+    assert old_bribesProcessor == strat_proxy.bribesProcessor()
+    
+    assert strat_proxy.auraBalToBalEthBptMinOutBps() == 0
```
[the remark this answers is not in the document]
Comment (shuklaayush) [reply tests/upgrade/test_upgrade.py:63]: Done

File: tests/integration/test_strategy_permissions.py [hunk 3]
```diff
@@ -281,6 +281,8 @@ def test_check_manual_permissions(strategy, want, strategist, reward_distributor
         strategy.setBribesProcessor(rando, {"from": rando})
     with brownie.reverts("onlyGovernanceOrStrategist"):
         strategy.claimBribesFromHiddenHand(rando, [], {"from": rando})
+    with brownie.reverts("onlyGovernanceOrStrategist"):
+        strategy.setAuraBalToBalEthBptMinOutBps(0, {"from": rando})
```
[the remark this answers is not in the document]
Comment (shuklaayush) [reply tests/integration/test_strategy_permissions.py:285]: Done

**Pull request merged**